extensions/markdown/syntaxes/markdown.tmLanguage

```diff
@@ -658,6 +662,22 @@
 						</dict>
 					</array>
 				</dict>
+				<key>fenced_code_block_php</key>
+				<dict>
+					<key>begin</key>
+					<string>(^|\G)\s*([`~]{3,})\s*(php|php3|php4|php5|phpt|phtml|aw|ctp)\s*$</string>
+					<key>name</key>
+					<string>markup.fenced_code.block.markdown</string>
+					<key>end</key>
+					<string>(^|\G)(\2)\n</string>
+					<key>patterns</key>
+					<array>
+						<dict>
+							<key>include</key>
+							<string>text.html.php</string>
```

Comment:
It's the same as the scopeName inside php syntax file, but currently the inline highlighting doesn't work. Will look into it later on.